llvm/tools/llvm-exegesis/lib/LlvmState.cpp

```diff
@@ -45,6 +45,12 @@ Expected<LLVMState> LLVMState::Create(std::string TripleName,
   if (CpuName == "native")
     CpuName = std::string(sys::getHostCPUName());
 
+  if (CpuName.empty()) {
+    std::unique_ptr<MCSubtargetInfo> Empty_STI(
+        TheTarget->createMCSubtargetInfo(TripleName, "", ""));
+    CpuName = Empty_STI->getAllProcessorDescriptions().begin()->Key;
```

Comment:
This behavior seems weird. Is this guaranteed to be a basic CPU or could it be one with lots of features?

Reply:
It's not even guaranteed that we can run code on the host when jitting for this first CPU, so it feels weird to automatically use this.

Reply:
Removed this code